HomeWork/MyHomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+var greeting = "Hello, playground"
+let MyFirstName = "Vyacheslav"
+let MySecondName = "Kocherin"
+print (MyFirstName+" "+MySecondName)
```

Comment:
```suggestion
print(MyFirstName + " " + MySecondName)
```
Разделяй пробелами правильно. Привыкай к синтаксису правильному